Validation/EventGenerator/src/DQMHelper.cc

```diff
@@ -0,0 +1,28 @@
+#include "Validation/EventGenerator/interface/DQMHelper.h"
+
+
+DQMHelper::DQMHelper(DQMStore::IBooker *i):ibooker(i){
+
+}
+
+DQMHelper::~DQMHelper(){
+
+}
+
+MonitorElement* DQMHelper::book1dHisto(const std::string &name,const std::string &title,int n,double xmin,double xmax){
```

Comment:
Hello,
     I have update this. So it should be fixed now.
Ian
On 02/11/2015 12:33 PM, deguio wrote:
>
> In Validation/EventGenerator/src/DQMHelper.cc 
> <https://github.com/cms-sw/cmssw/pull/7618#discussion_r24488654>:
>
> > @@ -0,0 +1,28 @@
> > +#include "Validation/EventGenerator/interface/DQMHelper.h"
> > +
> > +
> > +DQMHelper::DQMHelper(DQMStore::IBooker *i):ibooker(i){
> > +
> > +}
> > +
> > +DQMHelper::~DQMHelper(){
> > +
> > +}
> > +
> > +MonitorElement* DQMHelper::book1dHisto(const std::string &name,const std::string &title,int n,double xmin,double xmax){
>
> all these couts should be removed asap.
> @inugent <https://github.com/inugent> please make a PR to fix this.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/7618/files#r24488654>.
>